src/types.ts

```diff
@@ -14,3 +14,83 @@ export interface AddEthereumChainRequest {
   nativeCurrency?: NativeCurrencyData;
   rpcUrls?: string[];
 }
+
+export interface UpdateEthereumChainRequest {
```

Comment:
I'll suggest to add URL links of related EIPs/wallet docs from where these types are created.

Reply:
done